GSA-TTS/federal-website-standards
# Add baseurl to sitemap <loc>

## Context
This adds the base URL to our sitemap URLs for Search.gov. Fixes #162 

## Description
Prior to this change our [sitemap](https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/site/gsa-tts/federal-website-standards/sitemap.xml) had `<loc>` entries like this:

```xml
<url>
  <loc>/site/gsa-tts/federal-website-standards/404/</loc>
  <lastmod>2024-09-10</lastmod>
</url>
```

After this change, our `<loc>` entries will have the federalist URL prepended to them and look like this:

```xml
<url>
  <loc>https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/site/gsa-tts/federal-website-standards/404/</loc>
  <lastmod>2024-09-10</lastmod>
</url>
```

## How to verify this change
I'll need to track down the preview URL for this PR and I'll paste it here once I have it. To verify:

1. Visit the [sitemap](https://federalist-142078a8-f654-4daa-8f73-db9770b3ec7a.sites.pages.cloud.gov/preview/gsa-tts/federal-website-standards/caley/absolute-url-for-search/sitemap.xml) and verify that the federalist URL is a part of the `<loc>` values.
